EIPS/eip-2612.md

```diff
@@ -1,41 +1,42 @@
 ---
 eip: 2612
-title: permit – 712-signed approvals
-description: ERC-20 approvals via secp256k1 signatures
+title: "EIP-20 Permit Extension: Signed Approvals"
+description: EIP-20 approvals via EIP-712 secp256k1 signatures
 author: Martin Lundfall (@Mrchico)
 discussions-to: https://github.com/ethereum/EIPs/issues/2613
```

Comment:
Would you mind creating a new thread on Ethereum magicians?

Reply:
I explicitly asked about this at the beginning of this process and I was told it was okay to keep this link. Please do not put unnecessary hurdles on this PR that has been open for over a month already.

Reply:
This isn't a hurdle. This was truly a suggestion that would speed up the process later on (hence why I used a comment instead of a request for changes).

Reply:
Ok, sorry that I misunderstood.